src/usr/local/www/system_certmanager.php

```diff
@@ -163,21 +163,37 @@
 		/* Exporting a private key */
 		$keyout = base64_decode($thiscert['prv']);
 		if (isset($_POST['exportpass']) && !empty($_POST['exportpass'])) {
-			$res_key = openssl_pkey_get_private($keyout);
-			if ($res_key) {
-				openssl_pkey_export($res_key, $keyout, $_POST['exportpass']);
-			} else {
-				$savemsg = gettext("Unable to export password-protected private key.");
-				$class = 'danger';
-			}
-		}
+                        if ((strlen($_POST['exportpass']) < 4) or (strlen($_POST['exportpass']) > 1023)) {
```

Comment:
The indentation of all this needs fixed, it's using spaces and should be tabs

Reply:
hm, console copy/paste behavior
fixed